packages/server/src/modules/i18n/index.js

```diff
@@ -0,0 +1,16 @@
+import i18n from 'i18n';
+
+import Feature from '../connector';
+import settings from '../../../../../settings';
+
+const initMiddleware = async (req, res, next) => {
+  i18n.configure(settings.serverI18n);
```

Comment:
You should pass the language from cookie here to i18n and don't force developer to pass this cookie for every `translate` call. You are adding dumb work for the developer, avoid this at all costs.

Reply:
Not quite sure about this solution, since i18n is a global object and applying changes to it will lead to changes for all the clients who requesting the server at the same time.

Reply:
i18n is not a global object - it is a per-request object. Thats why you initialize this object in express middleware for each new HTTP/HTTPS request

Reply:
Your approach with using global object is wrong, it will have problems in production. You should redesign your approach to something like this:
https://github.com/i18next/i18next-express-middleware